dspace/config/spring/api/scripts.xml

```diff
@@ -102,4 +102,10 @@
        <property name="dspaceRunnableClass" value="org.dspace.app.suggestion.openaire.PublicationLoaderRunnableCli"/>
     </bean>
 
+    <bean id="import-loader-suggestions"
```

Comment:
Is this new script meant to **replace** the existing `import-openaire-suggestions` script?  To me, it seems like the `import-openaire-suggestions` script is no longer necessary.  Should we remove it and point at using `import-loader-suggestions` instead?